README.md

```diff
@@ -1,70 +1,58 @@
-# Getting Started with Create React App
+# Space Travelers' Hub
+This is a web application for a company that provides commercial and scientific space travel services. The application allows users to book rockets and join selected space missions.
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![Screenshot](src/assets/Screenshot1.png)
+![Screenshot](src/assets/Screenshot2.png)
+![Screenshot](src/assets/Screenshot3.png)
+## Built With
 
-## Available Scripts
+- HTML, CSS, JavaScript, React, and Webpack;
+- Visual Studio, Git & GitHub;
 
-In the project directory, you can run:
+## Getting Started
 
-### `yarn start`
+### Running locally
+To run this project, you only need a computer with a browser (like Google Chrome, Mozilla Firefox, Microsoft Edge, Apple Safari, etc.) installed, and follow these steps:
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+1. In your terminal, type the following, to clone this repository:
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+```sh
+$ git clone https://github.com/Chike1990/Space-Travelers-Hub
+```
 
-### `yarn test`
+2. After, run the following commands in your terminal:
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+```sh
+$ cd Space-Travelers-Hub
+$ npm install
+$ npm run build
+$ cd dist
+```
```

Comment:
- [ ] [OPTIONAL] You did a great job providing setup commands, but this command is not quite clear to non-technical users. For example, you did not specify what to be done after the user `cd` into the `dist` directory. Kindly give a clear explanation so that anyone can set up this project. Also, consider adding instructions to run the development server.